### PR TITLE
Narrow and wide campaign stuff.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
Replaces and is better than https://github.com/mozilla/foundation.mozilla.org/pull/1562

**Comment by ScottDowne** [comment]
@kristinashu OK, I cleaned up all the variant for the types of campaign pages. Links to all variants are here:

https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/wide-singlepage-petition/
https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/narrow-singlepage-petition/
https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/narrow-singlepage/
https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/wide-singlepage/
https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/wide-multipage-petition/
https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/narrow-multipage-petition/
https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/narrow-multipage/
https://foundation-mofostaging-pr-1568.herokuapp.com/campaigns/wide-multipage/

**Comment by ScottDowne** [comment]
Fixes https://github.com/mozilla/foundation.mozilla.org/issues/1564

File: network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html [hunk 1]
```diff
@@ -21,16 +23,22 @@ <h1>{{ block.value }}</h1>
...
-          <a href="#cta-anchor" class="narrow-sticky-button btn">TAKE ACTION</a>
+    <div class="cta
+      {% if uses_menu == False and page.specific.narrowed_page_content == False %}
+        col-lg-4 offset-lg-1
+      {% else %}
+        col-lg-12
+      {% endif %}"
+    >
+      <div class="narrow-sticky-button-container hidden">
+        <a href="#cta-anchor" class="narrow-sticky-button btn">TAKE ACTION</a>
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html]: Is this link supposed to exist on every campaign page?

**Comment by Pomax** [comment]
This looks pretty good, but I'm not sure about that hidden but always present "TAKE ACTION" link, is that really supposed to be on every single campaign page?

Also, do we need corresponding changes to the opportunity pages for the set of of narrow/wide, single/multi combinations?

File: network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html (hunk 1 above)
Comment (ScottDowne) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html]: yeah, (from my understanding) `if page.cta != None` would mean there is a useful place to send people if they click that button.

Comment (ScottDowne) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html]: Any campaign page without a cta, there shouldn't be a button.

**Pull request merged**